src/pages/signup.scss

```diff
@@ -0,0 +1,29 @@
+.log {
+  background-color: #f3f3f3 !important;
+  height: 400px;
+  margin-top: 60px;
+}
+.social {
+  margin-top: 20%;
+}
+.orr {
+  background-color: #f3f3f3 !important;
+  position: absolute;
+  top: 50%;
+  border-radius: 50%;
+  border: 1px solid #f3f3f3;
+}
+.verline {
+  border-right: 2px solid #dddddd;
+  height: 110px;
+  position: absolute;
+  top: 130px;
+  left: 50px;
+}
+.verline2 {
+  border-right: 2px solid #dddddd;
+  height: 110px;
+  position: absolute;
+  top: 65%;
+  left: 50px;
+}
```

Comment:
you are using a lot of properties that you can achieve similar thing with tailwind if you can use more tailwind and less css